app/bundles/DashboardBundle/Tests/Controller/DashboardControllerFunctionalTest.php

```diff
@@ -36,46 +36,14 @@ public function testWidgetWithReport(): void
         $this->em->flush();
         $this->em->detach($widget);
 
-        $this->client->xmlHttpRequest('GET', sprintf('/s/dashboard/widget/%s', $widget->getId()));
-        $this->assertResponseIsSuccessful();
+        $this->client->request('GET', sprintf('/s/dashboard/widget/%s', $widget->getId()), [], [], [
+            'HTTP_X-Requested-With' => 'XMLHttpRequest',
+        ]);
 
-        $content = $this->client->getResponse()->getContent();
-        Assert::assertJson($content);
+        $response = $this->client->getResponse();
+        Assert::assertSame(200, $response->getStatusCode());
 
-        $data = json_decode($content, true);
-        Assert::assertIsArray($data);
-        Assert::assertArrayHasKey('success', $data);
-        Assert::assertSame(1, $data['success']);
-        Assert::assertArrayHasKey('widgetId', $data);
-        Assert::assertSame((string) $widget->getId(), $data['widgetId']);
-        Assert::assertArrayHasKey('widgetWidth', $data);
-        Assert::assertSame($widget->getWidth(), $data['widgetWidth']);
-        Assert::assertArrayHasKey('widgetHeight', $data);
-        Assert::assertSame($widget->getHeight(), $data['widgetHeight']);
-        Assert::assertArrayHasKey('widgetHtml', $data);
-        Assert::assertStringContainsString('View Full Report', $data['widgetHtml']);
-    }
-
-    public function testWidgetWithBestHours(): void
```

Comment:
Any reason why this test was removed? I added it a few days ago. It seems like I wasted my time.

Reply:
Sorry, I made a mistake when making the changes for the PR to be sent to 5.2. Since I started it from the 5.x branch, it ended up updating with recent commits. I have removed them.